### PR TITLE
[DEVOPS-429] travis timeouts

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

**Comment by domenkozar** [comment]
cc @jmitchell for monitoring fixes.

File: modules/datadog-monitors.nix [hunk 1]
```diff
@@ -48,14 +48,15 @@ rec {
   mkMonitor = { name, type, query, message ? pagerDutyPolicy.normal, monitorOptions ? {} }:
     { config, ... }:
     baseMonitor // {
-      inherit name type message;
+      inherit type message;
+      name = "${config.deployment.name}: ${name}";
```
Comment (deepfire) [inline modules/datadog-monitors.nix]: Does this belong to a separate PR?

File: modules/datadog-monitors.nix [hunk 2]
```diff
@@ -48,14 +48,15 @@ rec {
...
       query = query config;
       monitorOptions = builtins.toJSON (baseMonitor.monitorOptions // monitorOptions);
     };
 
   # Common monitors
   cpu_monitor = {
-    name = "${config.deployment.name}: High CPU usage";
+    name = "High CPU usage";
```
Comment (deepfire) [inline modules/datadog-monitors.nix]: ditto

File: modules/datadog-monitors.nix (hunk 1 above)
Comment (domenkozar) [reply modules/datadog-monitors.nix]: This fixes evaluation, which was broken previously due to CI failing.

Comment (deepfire) [reply modules/datadog-monitors.nix]: Understood, thanks!

**Comment by deepfire** [comment]
What I see in this PR:

1. the git commit audit call was moved from `.travis.yml` to `travis.sh`
2. `travis.sh` now has Nixpkgs set
3. `travis.sh` now has a `travis_wait 30 sleep 1800 &` guard against timeouts
4. DataDog monitor names and definitions were slightly tweaked

Looks good to me.
